quickvision/models/detection/faster_rcnn/engine.py

```diff
@@ -262,8 +284,28 @@ def val_sanity_fit(model, val_loader,
                                  Useful is data is too big even for sanity check.
         log_interval : (optional) Defualt 100. Integer to Log after specified batch ids in every batch.
     """
-    pass
+    model = model.to(device)
+    model.eval()
 
+    cnt = 0
+
+    with torch.no_grad():
+        for batch_idx, (inputs, targets) in enumerate(val_loader):
+            
+            images = list(image.to(device) for k,v in t.items()} for t in targets)
+            targets = [{k: v.to(device) for k, v in t.items()} for t in targets]
+            out = model(images)
+
+            cnt += 1
+
+            if num_batches is not None:
+                if cnt >= num_batches:
+                    print(f"Done till {num_batches} validation batches")
+                    break
+
+            
+    
+    return True
 
```

Comment:
Same changes as above.
Not calculating metrics, or `AverageMeter` is fine.

Make CI pass, and add tests. Once we do that, the same code can be duplicated for retinanet.

Reply:
Noted